contracts/src/v0.8/dev/ocr2dr/OCR2DROracle.sol

```diff
@@ -24,24 +27,24 @@ contract OCR2DROracle is OCR2DROracleInterface, AuthorizedReceiver, ConfirmedOwn
 
   uint256 private constant MINIMUM_CONSUMER_GAS_LIMIT = 400000;
 
-  bytes32 private s_donPublicKey;
+  bytes private s_donPublicKey;
   uint256 private s_nonce;
   mapping(bytes32 => Commitment) private s_commitments;
 
-  constructor(address owner, bytes32 donPublicKey) ConfirmedOwner(owner) {
+  constructor(bytes memory donPublicKey) OCR2Base(true) {
```

Comment:
Can we have an owner-only setter for public key instead of providing it in the constructor? We should be able to rotate keys without re-deploying the contract.

Reply:
Good idea, done